pyvista/plotting/lights.py

```diff
@@ -11,7 +11,8 @@
 except ImportError:  # pragma: no cover
     from vtk import vtkLight, vtkLightActor, vtkMatrix4x4
 
-from ..utilities.helpers import vtkmatrix_from_array
+from pyvista.core.utilities.arrays import vtkmatrix_from_array
```

Comment:
I worked to implement a policy for absolute vs. relative imports (let me know what you think of this):

- If "this" module or lower, use relative imports. E.g., `from ._property import Property` in `pyvista/plotting/actor.py` or `from .utilities import blah`
- Otherwise, use absolute imports. E.g., `from pyvista.core.utilities.arrays import `vtkmatrix_from_array` in this file.

Why? 

- To avoid namespace conflicts since there is a `utilities` module in both `pyvista.core` and `pyvista.plotting` now.
- Because I plan to have follow up work that will actually split the `pyvista.core` and `pyvista.plotting` modules into totally separate packages and having the imports already split here will ultimately make this easier.
- For complete clarity. This splits the imports at the top level of the file and developers can easily distinguish what is coming from another part of the API and what is coming from "this" part of the API. Helping clarity what is a "core" utility vs a "plotting" utility

Reply:
> sounds like the trump here. No point doing anything else if relative imports would be broken when that split happens.

Indeed the trump card I'll play here

But also, I want to highlight another justification which I'm paraphrasing from someone else I was talking to about this:

> typically an `__init__.py` will import things below it (to re-expose them at the higher level). If some of these child modules want to import a sibling, it will have to go through the parent `__init__.py`, which causes a fragile partial import situation, which often outright breaks (depending on the order of imports within `__init__.py`, which is not normally something you care about, particularly when using isort ). So, the exception is that you should use relative imports for siblings and children sub-modules only.

I'm pretty sure since we weren't following this, its the reason PyVista's imports were so fragile and tended to become circular